### PR TITLE
Don't allow deps with no discernible license

### PR DESCRIPTION
**Pull request by thockin**
This updates the few deps we had with no LICENSE file to current versions that do have that file.  It also disallows new deps without obvious licenses.

### CODE REVIEW

File: hack/update-godep-licenses.sh [hunk 1]
```diff
@@ -141,21 +145,22 @@ for PACKAGE in $(cat Godeps/Godeps.json | \
...
-    content="${CONTENT[${PACKAGE}-COPYRIGHT]-}"
+      file="${CONTENT[${PACKAGE}-COPYRIGHT]-}"
   fi
-  if [[ -z "${content}" ]]; then
-      echo "UNKNOWN"
-      content="/dev/null"
-  else
-      cat "${content}"
+  if [[ -z "${file}" ]]; then
+      echo "Could not find a license for ${PACKAGE} - aborting" > /dev/stderr
```
Comment (david-mcmahon) [inline hack/update-godep-licenses.sh]: Do we want to provide anymore guidance here?  Links to any doc?

Comment (david-mcmahon) [inline hack/update-godep-licenses.sh]: godep could not locate a license/copyright for ${PACKAGE}.  

Options:
1. Contact the author of the package and ensure there is a license and godep can include it correctly
2. Do not submit this package to the Kubernetes tree.

Comment (thockin) [inline hack/update-godep-licenses.sh]: meh.  Give me better words and I'll stick them in.  or not.

Comment (david-mcmahon) [inline hack/update-godep-licenses.sh]: You don't like those?

**Comment by thockin** [comment]
re-pushed, actually builds now.  Let's see about tests.

**Comment by thockin** [comment]
Your words are fine.  Will fix next week

**Comment by thockin** [comment]
words added, rebased, re-pushed

**Pull request merged**